passes/hierarchy/hierarchy.cc

```diff
@@ -833,7 +833,7 @@ struct HierarchyPass : public Pass {
 		}
 
 		if (top_mod == nullptr && auto_top_mode) {
-			log_header(design, "Finding top of design hierarchy..\n");
+			log_header(design, "Finding top of design hierarchy...\n");
```

Comment:
There's a pretty consistent use of `..` instead of `...` at the end of log messages like this in Yosys. See `git grep -E '[^.]\.\.\.(\\n)?"'` vs `git grep -E '[^.]\.\.(\\n)?"'`.

We can have a discussion if it should be `...`, but I'm against just changing it here and nowhere else.

Reply:
I didn't realize it wasn't a typo. Removed that commit.